integration-tests/http/__tests__/order/admin/transfer-flow.spec.ts

```diff
@@ -141,6 +141,54 @@ medusaIntegrationTestRunner({
         expect(finalOrderResult.customer_id).toEqual(customer.id)
       })
 
+      it("should cancel an order transfer request from admin successfully", async () => {
+        await api.post(
+          `/admin/orders/${order.id}/transfer`,
+          {
+            customer_id: customer.id,
+          },
+          adminHeaders
+        )
+
+        await api.get(
+          `/admin/orders/${order.id}?fields=+customer_id,+email`,
+          adminHeaders
+        )
+
+        let orderPreviewResult = (
+          await api.get(`/admin/orders/${order.id}/preview`, adminHeaders)
+        ).data.order
+
+        expect(orderPreviewResult).toEqual(
+          expect.objectContaining({
+            customer_id: customer.id,
+            order_change: expect.objectContaining({
+              change_type: "transfer",
+              status: "requested",
+              requested_by: user.id,
+            }),
+          })
+        )
+
+        await api.post(
+          `/admin/orders/${order.id}/transfer/cancel`,
+          {},
+          adminHeaders
+        )
+
+        orderPreviewResult = (
+          await api.get(`/admin/orders/${order.id}/preview`, adminHeaders)
+        ).data.order
+
+        expect(orderPreviewResult.order_change).not.toBeDefined()
```

Comment:
Should we check this as well?

```suggestion
        expect(orderPreviewResult.order_change).not.toBeDefined()
        expect(orderPreviewResult.customer_id).not.toBeDefined()
```

Reply:
but customer_id of the original customer should be defined here, right?

Reply:
ah yeah, in that case expect it to be the old customer or not to be the new customer 🤞🏻